pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -4,6 +4,7 @@ import (
 	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
these two imports are the same, please remove the added line.

Reply:
done here 014f4b9